designer/server/src/main/scala/pl/touk/nussknacker/ui/NussknackerBoot.scala

```diff
@@ -0,0 +1,14 @@
+package pl.touk.nussknacker.ui
+
+import cats.effect.{ExitCode, IO, IOApp}
+import pl.touk.nussknacker.ui.factory.NussknackerApp
+
+object NussknackerBoot extends IOApp {
```

Comment:
We have fixed IDEA configurations in `.run/*.xml`, in scripts with entrypoints and also could be used in some dependant projects. IMO would be better to keep the old name and instead change extracted by you class to e.g. `NussknackerAppFactory`. WDYT?